pom.xml

```diff
@@ -30,7 +30,7 @@ THE SOFTWARE.
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.58</version>
+        <version>4.63</version>
```

Comment:
`4.64` is available - any reason this is not to be used?

Reply:
It wasn't available when I created this PR :) - 4 days ago. I updated it now